BlockServer/site_specific/default/general_rules.py

```diff
@@ -29,7 +29,7 @@
 
 """Standard Error message template for when regex for PV like names failes.
 Usage REGEX_ERROR_TEMPLATE_PV_NAME.format(<object name>)"""
-REGEX_ERROR_TEMPLATE_PV_NAME = "{0} must start with a letter and only contain letters, numbers and underscores"
+REGEX_ERROR_TEMPLATE_PV_NAME = "{0} must start with a letter and only contain letters, numbers and underscores, and be less than 25 characters long."
```

Comment:
Is that "25 characters or less" rather than "less than 25 characters"?

Reply:
It is 25 or less, I will change that now!